articles/openshift/oshift-how-expose-nonhttp-services.md

```diff
@@ -0,0 +1,72 @@
+---
+title: How to expose non-http services externally | UKCloud Ltd
+description: Explains how to expose services outside of your OpenShift cluster without using HTTP or HTTPS
+services: openshift
+author: Kieran O'Neill
+
+toc_rootlink: How To
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Expose Non-HTTP Services Externally
+toc_fullpath: How To/oshift-how-expose-nonhttp-services.md
+toc_mdlink: oshift-how-expose-nonhttp-services.md
+---
+
+# How to expose non-http services externally
+
+## Overview
+
+UKCloud for OpenShift enables you to develop, deploy, and manage digital and container-based applications seamlessly across local physical or virtual environments, with full portability to and from UKCloud.
+
+This guide explains how you can expose services outside of your OpenShift cluster without using routes.
+
+### Intended audience
+
+This guide assumes familiarity with services and routes in OpenShift and access to an account with cluster-admin permissions.
+
+### Prerequisites
+
+To complete the steps in this guide, your cluster must have been deployed from a version of our code which enables our non-http functionality. You can raise a support ticket to: check if this is already the case, to check if we can pull this version into your existing cluster or otherwise to request a new cluster be deployed incorporating our non-http functionality.
```

Comment:
...version of our code that enables...

Reply:
To make the options clearer, it might be better to put them in a bulleted list:

- Check if your cluster already provides non-http functionality

- Check if we can pull the non-http version of our code into your existing cluster

- Request a new cluster to be deployed incorporating our non-http functionality.

(You must include an empty line after each bullet item to make sure the HTML renders correctly)